### PR TITLE
Landscape app

### PR DESCRIPTION
**Pull request by kelkurniawan**
Changes : 
1. Fix JSON problems 
2. CSS
3. TDD
4. setup storybook

Story : https://www.pivotaltracker.com/story/show/181690072

### CODE REVIEW

File: src/components/watch-app/Watch.jsx [hunk 1]
```diff
@@ -19,16 +19,17 @@ function Watch() {
         })
         .then(response => response.json())
         .then(response => {
+            console.error(response);
```
Comment (KevinKP17) [inline src/components/watch-app/Watch.jsx]: kalau tidak dipakai hapus aja ya

File: src/components/watch-app/Watch.test.jsx [hunk 2]
```diff
@@ -7,9 +7,9 @@ import WatchStories from "./Watch.stories";
 import Watch,{ timeHelper} from "./Watch";
 
 const server = setupServer(
-    rest.get('./data/watchData.json', (req, res, ctx) => {
-        return res(ctx.json(watchData))
-    })
+    // rest.get('./data/watchData.json', (req, res, ctx) => {
```
Comment (KevinKP17) [inline src/components/watch-app/Watch.test.jsx:10]: comment kalau ga dipakai hapus aja ya

File: src/components/watch-app/Watch.test.jsx [hunk 3]
```diff
@@ -60,17 +60,25 @@ test("can display watchDuration when rendering the components from json", async(
...
-    expect(progressBar).toBeVisible()
-})
+//     const progressBar = await screen.findByDisplayValue("progressBar");
+//     expect(progressBar).toBeVisible()
+// })
 
-//test 6
-test.only("time helper will return formated duration when parameters in seconds", async() => { 
+//test 5
+test("time helper will return formated duration when parameters in seconds", async() => {
```
Comment (KevinKP17) [inline src/components/watch-app/Watch.test.jsx]: coba testnya yang lebih edge case

File: public/data/landscapeData.json [hunk 4]
```diff
@@ -0,0 +1,168 @@
+{
```
Comment (KevinKP17) [inline public/data/landscapeData.json:1]: ini data kan sama semua ya untuk landscape disatuin aja ya

File: public/data/watchData.json [hunk 5]
```diff
@@ -0,0 +1,168 @@
+{
```
Comment (KevinKP17) [inline public/data/watchData.json:1]: ini bedanya apa? ga ada beda satuin aja

File: src/components/landscape-exercise/landscapeData.jsx [hunk 6]
```diff
@@ -0,0 +1,168 @@
+const landscapeData = {
```
Comment (KevinKP17) [inline src/components/landscape-exercise/landscapeData.jsx:1]: ini bedanya apa? dijadikan 1 aja, truth data yang saya berikan kemarin aja di commentPT

File: src/components/watch-app/Watch.jsx [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+                        </div>
+                    </div> 
+                </>
+            )
+        })}
+    </div>
+    )
+}
+
+// export function timeHelper(duration){
```
Comment (KevinKP17) [inline src/components/watch-app/Watch.jsx:97]: comment hapus

File: src/components/landscape-exercise/Lancscape-app.test.jsx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+import { setupServer } from "msw/node"
+import LandscapeStories from "./Landscape.stories"
+import Landscape from "./lanscape-app"
+// Acceptance Criteria
+// can display components from json based on variations
+// landscape cards has progress bar calculated from watch_duration / duration * 100 %
+
+//============================================================================================================================================================
+const server = setupServer(
+    rest.get('./data/landscapeData.json', (req, res, ctx) => {
```
Comment (krowter) [inline src/components/landscape-exercise/Lancscape-app.test.jsx:17]: Ini pas setupServer ngga perlu kasih param kali sptnya. Soalnya di server.use di bawah udah ada ...LandscapeStories.parameters.msw nya
